easybuild/easyblocks/c/clang.py

```diff
@@ -552,20 +614,26 @@ def sanity_check_step(self):
         if self.cfg['static_analyzer']:
             custom_paths['files'].extend(["bin/scan-build", "bin/scan-view"])
 
-        if self.cfg['build_extra_clang_tools'] and LooseVersion(self.version) >= LooseVersion('3.4'):
+        if 'clang-tools-extra' in self.cfg['llvm_projects'] and LooseVersion(self.version) >= LooseVersion('3.4'):
             custom_paths['files'].extend(["bin/clang-tidy"])
 
-        if self.cfg["usepolly"]:
+        if 'polly' in self.cfg['llvm_projects']:
             custom_paths['files'].extend(["lib/LLVMPolly.%s" % shlib_ext])
             custom_paths['dirs'].extend(["include/polly"])
 
-        if self.cfg["build_lld"]:
+        if 'lld' in self.cfg['llvm_projects']:
             custom_paths['files'].extend(["bin/lld"])
 
-        if self.cfg["libcxx"]:
+        if 'lldb' in self.cfg['llvm_projects']:
+            custom_paths['files'].extend(["bin/lldb"])
+
+        if 'libcxx' in self.cfg['llvm_projects']:
             custom_paths['files'].extend(["lib/libc++.%s" % shlib_ext])
             custom_paths['files'].extend(["lib/libc++abi.%s" % shlib_ext])
 
+        if 'flang' in self.cfg['llvm_projects'] and LooseVersion(self.version) >= LooseVersion('15'):
+            custom_paths['files'].extend(["bin/flang"])
```

Comment:
```suggestion
            custom_paths['files'].extend(["bin/flang-new"])
```
Sorry, my mistake, right now that compiler is called `flang-new`

Reply:
It will change at some point to `flang` (but I can't find a timeline)

Reply:
`flang-new` is the one recognised by CMake 3.24.0+